seatunnel-dist/release-docs/LICENSE

```diff
@@ -693,6 +693,7 @@ The text of each license is the standard Apache 2.0 license.
      (The Apache Software License, Version 2.0) Commons Logging (commons-logging:commons-logging:1.1.3 - http://commons.apache.org/proper/commons-logging/)
      (Apache 2) Objenesis (org.objenesis:objenesis:2.5.1 - http://objenesis.org)
      (The Apache Software License, Version 2.0) Apache Commons Math (org.apache.commons:commons-math3:3.5 - http://commons.apache.org/proper/commons-math/)
+     (The Apache Software License, Version 2.0) Apache SkyWalking : 8.9.0 (https://github.com/apache/skywalking/)
```

Comment:
Do you include this script into the binary tar? If not, this license doesn't need to update.

Reply:
yup, we don't need to keep this in binary tar, removed